bigquery/tests/unit/test__helpers.py

```diff
@@ -765,7 +765,7 @@ def _get_target_class():
     def test_it(self):
 
         class Sub(self._get_target_class()):
-            ALLOWED = ('FOO', 'BAR', 'BAZ')
+            pass
```

Comment:
Without `_validate`, there really isn't any point in the `_EnumProperty` base class at all:  the concrete classes could just derived from `_ConfigurationProperty, leaving the class-level constants (but then, why, if we aren't going to enforce them?)